src/intl/en/common.json

```diff
@@ -123,6 +123,7 @@
   "language-ko": "Korean",
   "language-lt": "Lithuanian",
   "language-ml": "Malayalam",
+  "language-mr": "Marathi",
```

Comment:
Adding Marathi at this stage means the languages page [is out of date immediately](https://ethereumorgwebsitedev01-mrhomepage.gtsb.io/mr/languages/)